### ISSUE TITLE
[Chore] Migrate code gen out of recorder and use Synthetics again

### ISSUE THREAD

**Issue by justinkambic** [elastic/synthetics-recorder#407]
### Task summary

We had to move code gen into the recorder [a while back](https://github.com/elastic/synthetics-recorder/pull/211) because we were blocked on upgrading synthetics/playwright. We are unblocked on that now and there's no reason to have this code split into the recorder, so we should migrate back to using Synthetics code gen.

### Linked resources

It [was asked](https://github.com/elastic/synthetics-recorder/issues/393) as part of our recent playwright upgrade, but we [didn't do that](https://github.com/elastic/synthetics-recorder/issues/393#issuecomment-1513516765).

**Comment by justinkambic** [comment]
Closing this in favor of the existing https://github.com/elastic/synthetics-recorder/issues/295.
